app-modules/knowledge-base/graphql/knowledge-base-item.graphql

```diff
@@ -0,0 +1,12 @@
+type KnowledgeBaseItem
+    @feature(feature: "knowledge-management")
+    @model(class: "AdvisingApp\\KnowledgeBase\\Models\\KnowledgeBaseItem") {
```

Comment:
This seems to be the only `@model` that uses the FQN. Was there a reason for this or can it be switched to `knowledge_base_item`?

Reply:
For some reason it would not resolve the morphmap. Perhaps something with it being snake_case. Right now I think our best option is to define all the models with FQN. That is what I am doing in my current branch, and we can update the others to do the same.

Reply:
Perhaps we can figure out the morphmap thing later. But defining the FQN on the Model I think is sufficient for now. Spending time trying to get Morphmap to work would not be worth the effort atm.